tests/playwright/src/utility/fixtures.ts

```diff
@@ -30,21 +30,29 @@ export type TestFixtures = {
   navigationBar: NavigationBar;
   welcomePage: WelcomePage;
   page: Page;
+  context: BrowserContext;
   statusBar: StatusBar;
 };
 
 export type FixtureOptions = {
   runnerOptions: RunnerOptions;
 };
 
+let newContext = false;
```

Comment:
This is anti-pattern and doesn't work, since `newContext` will always be true when checked by `page` fixture, because `context` fixture will be called before the page fixture due to the dependency chain defined in this PR.